### ISSUE TITLE
Cannot read property 'this._transport._transport' of undefined

### ISSUE THREAD

**Issue by dragnet** [centrifugal/centrifuge-js#74]
You have a mistake in conditions:
```    
const transportOpen = this._transport &&
      this._transport._transport &&
      this._transport._transport.readyState === this._transport._transport.OPEN;
```
`this._transport._transport` of undefined

Need change to:
```    
const transportOpen = this._transport && this._transport.readyState === this._transport.OPEN;
```

**Comment by FZambia** [comment]
@dragnet thanks a lot!

The reason is in difference between SockJS and Websocket - I've tested changes using only SockJS transport. Will try to add more tests in future but just don't have enough time for this :(

I believe that the problem should be fixed by 4102f04a95f6d258ef71c91affc6534d4e8ef0ea, already released 2.1.1 with this change. Could you try it and check that it works properly for you now?

**Comment by dragnet** [comment]
yes, it works correct for now, thanks, Alexander, you reacted very quickly
